solidity/scripts/lcl-client-config.js

```diff
@@ -44,7 +44,25 @@ module.exports = async function () {
       const formattedConfigFile = tomlify.toToml(fileContent, {
         space: 2,
         replace: (key, value) => {
-          return key == "Port" ? value.toFixed(0) : false
+          let result
+          try {
+            result =
+              // We expect the config file to contain arrays, in such case key for
+              // each entry is its' index number. We verify if the key is a string
+              // so we can run the following match check.
+              typeof key === "string" &&
+              // Find keys that match exactly `Port` or ends with `MetricsTick`.
+              key.match(/(^Port|MetricsTick)$/)
```

Comment:
We have two other numerical properties that we also need to handle here: `MiningCheckInterval` and `MaxGasPrice`.

Reply:
I tested with `key.match(/(^Port|MetricsTick|MiningCheckInterval|MaxGasPrice)$/)` and everything works as expected.